src/App.js

```diff
@@ -1,16 +1,33 @@
 import "./App.css";
+import 'react-toastify/dist/ReactToastify.css';
+
 import { Routes, Route } from "react-router-dom";
-import { LandingPage } from './Pages';
-import { Home} from "./Pages/index";
+import {ToastContainer} from 'react-toastify';
+import Mockman from 'mockman-js';
+
+import { LandingPage, Home, Login, Signup } from './Pages';
 import { Navbar } from "./component";
 
 function App() {
   return (
     <div className="App">
+       <ToastContainer 
+        position="top-right"
+        autoClose={2000}
+        hideProgressBar={false}
+        newestOnTop={false}
+        closeOnClick
+        rtl={false}
+        draggable
+        pauseOnHover
+      />
       <Navbar />
       <Routes>
-        <Route path='/' element={<LandingPage />}/>
-        <Route path='/home' element={<Home />}/>
+        <Route path='/' element={<LandingPage />} />
```

Comment:
You can add an error 404 page. Can refer [this](https://ui.dev/react-router-handling-404-pages)

Reply:
Will surely explore this. Thanks for the link though.